source/help/messaging/executing-commands.md

```diff
@@ -0,0 +1,18 @@
+# Executing Commands
+___
+Slash commands allow users to perform operations in Mattermost or interact with external applications by typing into the text input box. A user will enter a `/` followed by a command and some arguments to perform actions. There are built-in slash commands that come with all Mattermost installations, and optionally, custom slash commands may be configured to interact with external applications. To learn more about configuring custom slash commands, please visit the [developer slash command documentation page](../developer/slash-commands.md).
```

Comment:
Propose trying a [more concise phrasing?](http://docs.mattermost.com/process/documentation-guidelines.html#star-requirements-subjective-style-guidelines) Could probably lose 20%+ of words without losing information.